sdcm/cluster.py

```diff
@@ -721,6 +721,23 @@ def wait_db_up(self, verbose=True):
         wait.wait_for(func=self.db_up, step=60,
                       text=text)
 
+    def apt_not_running(self):
```

Comment:
It seems more straight forward to make an `apt_running` function, then use a `lambda: not self.apt_not_running` as a parameter for `wait.wait_for`.